interpreter/cling/lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -91,6 +91,19 @@ std::unique_ptr<TargetMachine>
   CodeModel::Model CMModel = CodeModel::JITDefault;
   CodeGenOpt::Level OptLevel = CodeGenOpt::Less;
 
+  const char* OptLevelEnvConfig = std::getenv("ROOT_OPTIMIZATION_LEVEL");
```

Comment:
That sounds much better indeed.
I need to look deeper into the code to achieve that though.

Just to be clear, you're suggesting to pass the `-Ox` flags to the `args` of
`cling::Interpreter`'s constructor, and then using it here?